pkg/actors/reminders/reminders.go

```diff
@@ -159,9 +160,19 @@ func (r *reminders) CreateReminder(ctx context.Context, reminder *internal.Remin
 
 	stop := make(chan struct{})
 
-	err = r.storeReminder(ctx, store, reminder, stop)
+	config := retry.DefaultConfig()
```

Comment:
Does this have a maximum attempt count?

Reply:
no, I skipped it intentionally to ensure that we definitely store/delete reminder

Reply:
And, I kept it Multiplier as 1.0 intentionally, to ensure that workflows execution time doesn't exceed too much.